packages/backend/src/notifications.test.ts

```diff
@@ -90,18 +177,104 @@ describe('Notifications', () => {
                 getIdEnsName: async (ensName: string) => ensName,
                 addUsersNotificationChannel: addUsersNotificationChannelMock,
             };
+
             app.locals.web3Provider = {
                 resolveName: async () =>
                     '0x71CB05EE1b1F506fF321Da3dac38f25c0c9ce6E1',
             };
 
             const { status } = await request(app)
-                .post(`/email/bob.eth`)
+                .post(`/bob.eth`)
+                .set({
+                    authorization: `Bearer ${token}`,
+                })
+                .send({
+                    recipientValue: 'bob@gamil.com',
+                    notificationChannelType: '',
+                });
+
+            expect(status).toBe(400);
+        });
+
+        it('Returns 500 on setup email notifications as globalNotifications is turned off', async () => {
+            const app = express();
+            app.use(bodyParser.json());
+            app.use(notifications());
+
+            const token = await createAuthToken();
+            const addUsersNotificationChannelMock = jest.fn();
+
+            app.locals.db = {
+                getSession: async (ensName: string) =>
+                    Promise.resolve({
+                        challenge: '123',
+                        token,
+                    }),
+                setSession: async (_: string, __: any) => {
+                    return (_: any, __: any, ___: any) => {};
+                },
+                setUserStorage: (_: string, __: string) => {},
+                getIdEnsName: async (ensName: string) => ensName,
+                getGlobalNotification: async (ensName: string) =>
+                    Promise.resolve({ isEnabled: false }),
+                addUsersNotificationChannel: addUsersNotificationChannelMock,
+            };
+
+            app.locals.web3Provider = {
+                resolveName: async () =>
+                    '0x71CB05EE1b1F506fF321Da3dac38f25c0c9ce6E1',
+            };
+
+            const { status } = await request(app)
+                .post(`/bob.eth`)
+                .set({
+                    authorization: `Bearer ${token}`,
+                })
+                .send({
+                    recipientValue: 'bob@gamil.com',
+                    notificationChannelType: NotificationChannelType.EMAIL,
+                });
+
+            expect(status).toBe(500);
```

Comment:
Please use Code 404 instead along with an error message that the particular message is not supported yet